web/src/app/services/data-store/data-store.service.ts

```diff
@@ -313,7 +313,7 @@ export class DataStoreService {
   }
 
   getAccessibleLois$(
-    {id: surveyId, acl}: Survey,
+    {id: surveyId}: Survey,
```

Comment:
If we only need `surveyId`, let's pass that instead of requiring the whole `Survey` object to be passed.